### ISSUE TITLE
Incorrect linebreaks in the preabmble

### ISSUE THREAD

**Issue by newptcai** [cmhughes/latexindent.pl#298]
Please provide the following when posting an issue:

original .tex code
--
~~~
\documentclass[11pt]{article}

\usepackage{titlesec}
\titleformat{\section}{\raggedright\large\bfseries\color{dku-blue}}{}{0em}{}[{\titlerule[\heavyrulewidth]}]
\titleformat{\subsection}{\raggedright\itshape\color{dku-blue}}{}{0em}{}

\begin{document}

haha

\end{document}
~~~
yaml settings
--
~~~
# default value of indentation
defaultIndent: "    "

modifyLineBreaks:
    preserveBlankLines: 1
    condenseMultipleBlankLinesInto: 1
    oneSentencePerLine:
        manipulateSentences: 1
        removeSentenceLineBreaks: 1
        sentencesFollow:
            other: "\)|\}"
        sentencesEndWith:
            basicFullStop: 0
        textWrapSentences: 1
    textWrapOptions:
        columns: 85
        preamble: 0

verbatimEnvironments:
    longtable: 1

verbatimCommands:
    titleformat: 1
    titlerule: 1
    href: 1

fineTuning:
    modifyLineBreaks:
      betterFullStop: '(?:\.\)(?!\h*[a-z]))|(?:(?<!(?:(?:e\.g)|(?:i\.e)|(?:etc))))\.(?!(?:\\|[a-z]|[A-Z]|\-|~|\,|[0-9]))'
~~~

actual/given output
--
~~~
\documentclass[11pt]{article}

\usepackage{titlesec}
\titleformat{\section}{\raggedright\large\bfseries\color{dku-blue}}{}{0em}{}[{\titler
ule[\heavyrulewidth]}]
\titleformat{\subsection}{\raggedright\itshape\color{dku-blue}}{}{0em}{}

\begin{document}

haha

\end{document}
~~~

desired or expected output
--
It should be identical to the original tex file. The command `\titlerule` has been broken.

**Comment by cmhughes** [comment]
Is your yaml minimal? Anything you can do to help minimise the yaml would be helpful!

**Comment by newptcai** [comment]
This seems to have been fixed in version 3.13. I was using the version coming with texlive 2020.
